packages/backend/src/app.ts

```diff
@@ -84,6 +84,7 @@ export type ClientKeysContext = Context<AppRequest<'keyId'>>
 export interface PaymentPointerContext extends AppContext {
   paymentPointer: PaymentPointer
   grant?: Grant
+  clientId?: string
```

Comment:
Isn't that also a property of `grant`?

Reply:
Yes, but `ctx.clientId` is now being defined in the auth middleware if the request has limited access to client-specific resources.
(instead of checking the grant in every relevant route)
Maybe it'd be clearer if we replace `grant` here with `grantId`, but also add a `limits?: AccessLimits` (to be used in outgoing payment creation)?

Reply:
> Maybe it'd be clearer if we replace grant here with grantId, but also add a limits?: AccessLimits (to be used in outgoing payment creation)?
Let's do that.

Reply:
I started making this change but it seems like it'll be a bit extensive, so I'd prefer addressing it as a separate issue:
- https://github.com/interledger/rafiki/issues/688